.github/workflows/runtests.yml

```diff
@@ -0,0 +1,107 @@
+name: validate
+on: pull_request
+
+# Set globally scoped environmental variables.
+env:
+  ISSUE_NUMBER: ${{ github.event.pull_request.number }}
+  PR_URL: https://github.com/obspy/obspy/pull/${{ github.event.pull_request.number }}
+  CI_URL: https://github.com/obspy/obspy/actions/runs/${{ github.run_id }}
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+
+jobs:
+  # Parse the comments from the PR and uploads a json with selected options
+  get_ci_config:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+
+      - uses: actions/setup-python@v1
+        with:
+          python-version: '3.x'
+
+      - name: create ci config
+        shell: bash -l {0}
+        run: |
+          pip install git+https://github.com/d-chambers/obspy_github_api@gh-actions
```

Comment:
I guess you want in the end to point here to the original repo?

Reply:
Yes, but I don't have permission to write to it. I will submit a PR then we can change it.